opal/corelib/variables.rb

```diff
@@ -12,9 +12,9 @@
 $/ = "\n"
 $, = nil
 
-::ARGV = []
+::ARGV = `Opal.platform.argv`
+::ARGV.shift if ::ARGV.first == '--'
```

Comment:
Maybe this should be handled by a runner?

Reply:
Good catch, line 16 should be handled by platform driver.

Reply:
Well, when you compile a script and later run it, lets say with node or deno, there is no runner. So the driver should correctly identify script args.